src/prototypes/draft-email-textarea/example/example.twig

```diff
@@ -0,0 +1,64 @@
+<div class="o-container o-container--pad o-container--prose">
+	<div class="o-container__content o-rhythm o-rhythm--default o-rhythm--generous-headings">
+
+		<textarea id="email-draft" class="c-input u-pad-1 js-elastic-textarea" spellcheck="false" rows="5">Hi there,
+
+My name is [Firstname Lastname], I’m a [role] at [organization].
+
+I’m reaching out to discuss [project, question, challenge]. [Additional details, if any.]
+
+Our budget for this effort is [budget, range, etc.] and we’re interested in wrapping up in [timeframe].
+
+I’m available to chat [potential meeting times], let me know if any of those work for you.
+
+Have a great day!
+
+[Firstname Lastname]</textarea>
+
+		<div class="o-button-group o-button-group--grow u-space-block-start-n2">
+			<button id="copy-btn" class="c-button">
+				<span class="c-button__extra">
+					<svg width="24" height="24" viewBox="0 0 24 24" aria-hidden="true" class="c-icon">
+						<path d="M16 16v5c0 .55-.45 1-1 1H3c-.55 0-1-.45-1-1V9c0-.55.45-1 1-1h5" fill="none" stroke-linecap="round" stroke-width="2"></path>
+						<rect x="8" y="2" width="14" height="14" rx="1" ry="1" fill="none" stroke-linecap="round" stroke-width="2"></rect>
+					</svg>
+				</span>
+				<span class="c-button__content"> Copy </span>
+			</button>
+
+			<a href="mailto:info@cloudfour.com?subject=Discuss potential project&body=Hi there,%0D%0A%0D%0AMy name is [Firstname Lastname], I’m a [role] at [organization].%0D%0A%0D%0AI’m reaching out to discuss [project, question, challenge]. [Additional details, if any.]%0D%0A%0D%0AOur budget for this effort is [budget, range, etc.] and we’re interested in wrapping up in [timeframe].%0D%0A%0D%0AI’m available to chat [potential meeting times], let me know if any of those work for you.%0D%0A%0D%0AHave a great day!%0D%0A%0D%0A[Firstname Lastname]" class="c-button c-button--secondary">
```

Comment:
If you wanted to keep things DRY, you could pass the textarea content to the template from the story definition as a string, then output it once in the textarea, and once here.

Twig has a `url_encode` filter that comes in handy for things like this: https://twig.symfony.com/doc/3.x/filters/url_encode.html

Reply:
Thanks that sounds like an excellent idea! 

Though, I'm having a hard time wrapping my head around your suggestion within the context of the WP Gutenberg editor. 🤔